src/ui/component/collapsible/collapsible.tsx

```diff
@@ -0,0 +1,52 @@
+import type { FC } from 'react'
+import { useState, useCallback } from 'react'
+import classNames from 'classnames'
+import * as RCollapsible from '@radix-ui/react-collapsible'
+import type { IconName } from '@/ui/component/icon/icon'
+import { Icon } from '@/ui/component/icon/icon'
+import type { AssertSubset } from '@/util/type'
+import './collapsible.scss'
+
+type CollapsibleProps = {
+  children: JSX.Element
+  triggerElement: JSX.Element
```

Comment:
The naming is not consistent. For properties of type `JSX.Elment`, you need to decide whether the property name should be suffixed or not by `Element`.

Reply:
Good point! We can remove the `Element` suffix since the type `JSX.Element` already indicates it's an element. I suggest renaming `triggerElement` to `trigger` for consistency and simplicity.